src/api/apiClient.tsx

```diff
@@ -1,29 +1,29 @@
 import axios from 'axios';
-// export const NOTIFICATIONS_API_ROOT = '/api/xavier/camel';
-export const NOTIFICATIONS_API_ROOT = '/camel';
+export const NOTIFICATIONS_API_ROOT = '/api/xavier/camel';
```

Comment:
I commented this line (and de-commented the next one to have it working)
```suggestion
//export const NOTIFICATIONS_API_ROOT = '/api/xavier/camel';
```